insights/parsers/getconf.py

```diff
@@ -0,0 +1,21 @@
+"""
+getconf PAGE_SIZE - command ``/usr/sbin/getconf PAGE_SIZE``
+=============================================
+
+This very simple parser returns the output of the ``getconf PAGE_SIZE`` command.
+
+    >>> output = shared[getconfpagesize]
```

Comment:
An `Examples:` section header is necessary for these examples. 

See https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html